boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -399,4 +418,157 @@ TEST_F(
     }
 }
 
+// DCM-8
+TEST_F(
+    DcmStateMachineTest,
+    regen_allowed_only_when_going_faster_than_5kph_and_airs_closed)
+{
+    SetInitialState(App_GetDriveState());
+
+    // Turn the DIM start switch on to prevent state transitions in
+    // the drive state.
+    App_CanRx_DIM_SWITCHES_SetSignal_START_SWITCH(
+        can_rx_interface, CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE);
+
+    // Press down accelerator pedal and regen paddle
```

Comment:
The code already communicates this clearly so comment here isn't needed imo
```suggestion
```

Reply:
Are the pedal % and paddle position values chosen arbitrarily here?

Reply:
Yeah